unit/goto-programs/goto_program_generics/GenericFields.java

```diff
@@ -0,0 +1,99 @@
+// THIS FILE HAS A COPY IN test-gen/unit/test-java-gen
```

Comment:
does git allow for symbolic links? we don't have to support Windows in TG

Reply:
I removed this because the tests in test-gen will be changed to not be just a copy.